Marlin/src/lcd/language/language_en.h

```diff
@@ -173,7 +173,7 @@
   #define MSG_EXTRUDE                         _UxGT("Extrude")
 #endif
 #ifndef MSG_RETRACT
-  #define MSG_RETRACT                         _UxGT("Retract")
+  #define MSG_RETRACT                         _UxGT("FW Retract")
```

Comment:
i dunno... when user sees "Retract" in the menu i think he/she probably know this refers to the firmware retraction settings.
plus this adds 3 bytes to progmem....

Reply:
“Retract” to me implies “Perform a retraction immediately” not “enter retraction settings” though if that 3 bytes is so paramount it can be removed...

Reply:
I thought this menu item should be inside something line "Config" or "Settings" menu......

Reply:
It is, however for the sake of the Tune menu, which doesnt have a "Config" or "Settings" menu. I thought it wise to clarify it. If this is an issue though, ill remove those 3 bytes.